### PR TITLE
FAI-13016: Remove lerna and migrate to Turbo repo

### PR DESCRIPTION
**Pull request by chalenge**
## Description

- Bye bye lerna 

## Type of change
- [X] Bug fix
- [X] New feature
- [X] Breaking change

### CODE REVIEW

File: sources/agileaccelerator-source/README.md [hunk 1]
```diff
@@ -82,12 +82,3 @@ To run unit tests locally, from the AgileAccelerator connector directory run:
 ```
 npm test
 ```
-
-
-## Dependency Management
```
Comment (chalenge) [inline sources/agileaccelerator-source/README.md:87]: I think we should move one of these to the main section after [Development](https://github.com/faros-ai/airbyte-connectors/pull/1690/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L47)

File: turbo.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "test": {
+      "dependsOn": ["build"],
+      "inputs": ["test/**", "resources/**", "test_files/**"]
+    },
+    "test-cov": {
+      "dependsOn": ["build"],
+      "inputs": ["test/**", "resources/**", "test_files/**"]
+    }
+  }
+}
```
Comment (chalenge) [inline turbo.json:27]: nit newline

**Review by ypc-faros — APPROVED** [review]
🔥

**Comment by tovbinm** [comment]
Yay!!!

**Pull request merged**